.travis.yml

```diff
@@ -37,15 +37,19 @@ matrix:
         - python3 -m pip install -e sdk/python
       script:
         - make report
-    - name: "Verify source files contain the license header"
-      language: bash
-      script:
-        - make check_license
     - name: "Lint Python code with flake8"
       language: python
       python: "3.7"
       script:
         - make lint
+    - name: "Verify source files contain the license header"
+      language: bash
+      script:
+        - make check_license
+    - name: "Verify Markdown files have current table of contents"
+      language: bash
+      script:
+        - make check_mdtoc
```

Comment:
Can we group some these tests (e.g. license and mdtoc check) into one job before we start overwhelming the Travis CI quota?

Reply:
What is the quota we have with Travis? Until we hit a limit I would like to keep them separate. Maybe there is a way to share common setup between Travis jobs but still report separate logical jobs?

Reply:
Right now it should be fine. For open source I think we are allowed to run up to 200 jobs in parallel per repo. So if we have more jobs, less commits/PR can be tested in parallel.

Reply:
This is something to put in mind as we creating more tests. For now, it LGTM